avocado/core/remote/runner.py

```diff
@@ -45,6 +45,11 @@ class RemoteTestRunner(TestRunner):
     remote_version_re = re.compile(r'^Avocado (\d+)\.(\d+)$',
                                    re.MULTILINE)
 
+    def __init__(self, job, test_result):
+        super(RemoteTestRunner, self).__init__(job, test_result)
+        #: remoter connection to the remote machine
+        self.remote = None
```

Comment:
OK, I don't find it necessary for `None` initialization, but I have nothing against doing it this way...